Added `remove.md` in the terms folder of `hashmap` (#1552)
Codecademy/docs: content/kotlin/concepts/hashmaps/terms/remove.md

---
Title: '.remove()'

> Description: 'Used to remove a key-value pair from a HashMap'

Reviewer: ```suggestion
Description: 'Used to remove a key-value pair from a HashMap.'
```

Subjects:
  - 'Computer Science'
Tags:
  - 'HashMap'
  - 'Data Types'
  - 'Collections'
  - 'Data Structures'
CatalogContent:
  - 'learn-kotlin'
  - 'paths/computer-science'
---

> To remove a key-value pair from a `HashMap` in Kotlin, you can use the `remove()` function.

Reviewer: ```suggestion
The **`.remove()`** method is used to remove a key-value pair from a `HashMap` in Kotlin.
```

## Syntax

```pseudo
fun <K, V> MutableMap<K, V>.remove(key: K): V?
```

> It takes one argument:

Reviewer: The bulleted list makes it appear both are arguments. Instead of introducing a bulleted list like this, just have two sentences: "It takes one parameter..." and "It returns..."

- `key`: the key of the key-value pair to be removed
- `V?` : the value associated with the removed key or null if the key is not found in the map.

## Example

> Here is an example of how we can use the `remove()` function in a Kotlin program:

Reviewer: ```suggestion
Here is an example of how to use the `.remove()` method in a Kotlin program:
```
